Makefile

```diff
@@ -8,12 +8,25 @@ doc:
 	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
 	@echo
 
+doc3:
+	python3 setup.py build_sphinx
+	@echo
+	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
+	@echo
+
 test:
 	-find coverage/ -mindepth 1 -delete
 	python $$(which nosetests) $${TESTS}
 
+test3:
+	-find coverage/ -mindepth 1 -delete
+	python3 $$(which nosetests3) $${TESTS}
+
```

Comment:
Again, I have no knowledge of `tox`.  I did, however, use the `test3` target extensively and progressively test my changes in Python 3.